notebooks/road_following/data_collection.ipynb

```diff
@@ -237,15 +237,15 @@
     "def xy_uuid(x, y):\n",
```

Comment:
Can we change function name from xy_uuid to steering_throttle_uuid ? I dont see that change.

Reply:
I instead changed all references to ``steering_throttle_uuid`` to ``xy_uuid``.  I think this should resolve the issues.  I prefer ``xy_uuid`` because it's very clear (since x, y don't _exactly_ map to steering / throttle, and we don't perform throttling)

Reply:
Ok.